src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -96,8 +99,15 @@ export class ArtworkSidebarBidAction extends React.Component<
   redirectToBid(firstIncrement: number) {
     const { slug, sale } = this.props.artwork
     const bid = this.state.selectedMaxBidCents || firstIncrement
+
     const href = `/auction/${sale?.slug}/bid/${slug}?bid=${bid}`
-    window.location.href = href
+
+    // FIXME: Remove this after Auction2 launches
+    if (getENV("ENABLE_AUCTION_V2")) {
```

Comment:
Wondering if we could use the new feature flag service to roll this out? 🤔

Reply:
I think that service is still being worked on